### ISSUE TITLE
Radios require both passed in and chained value.

### ISSUE THREAD

**Issue by kynetiv** [netojose/laravel-bootstrap-4-forms#58]
I'm trying to set the default value of a radio with this (simplified) snippet by chaining `->value()`:

```
    {!! Form::radio("gender", "Gender")->value($value) !!}
```
Instead of passing in the value directly to the radio method. 

```
    {!! Form::radio("gender", "Gender", $value) !!}
```

I see in the FormService (line~892), that it is only checking for the passed in value.
```php
    private function _checkboxRadio($type, $name, $label, $value, $default): FormService
    {
        $inputValue = $value === null ? $name : $value;

        if ($default) {
            $default = $inputValue;
        }

        return $this->_set('meta', ['value' => $inputValue])->type($type)->name($name)->label($label)->value($default);
    }
```
The result of the chaining approach is that the radio value is set to the  form `name`. It doesn't later check if another value was provided by the chaining method.

Can this be supported? 

Perhaps if this `_checkboxRadio` was moved to the FormBuilder and there is access to _getValue() we could add another check there?

Thank you!

**Comment by kynetiv** [comment]
After working through this more, it looks like both a passed in and chained values are required (chained is needed for setting a previously selected value).

**Comment by netojose** [comment]
Fixed on 3.0.0 version. Please let me know if the issue still exists.
